device_doctor/lib/src/android_device.dart

```diff
@@ -117,7 +117,6 @@ class AndroidDeviceDiscovery implements DeviceDiscovery {
     final String propertiesJson = json.encode(properties);
 
     await writeToFile(propertiesJson, _outputFilePath);
-    stdout.write(propertiesJson);
```

Comment:
Would printing this out to stdout still be helpful? Maybe as a log message?

Reply:
Since writeTofile does exactly the same thing as the stdout.write, and from the bot_config side, we are not utilizing the stdout any more, this line doesn't help much.
Would prefer to do the cleanup now. If we need this at any point in the future, we can always add it back.

Reply:
Maybe a logging.info? if the properties are unexpected we won't have any way to verify the problem.

Reply:
We only have a client side call to the device_doctor, namely no server is running. I am not sure where the logging.info will be logged to?

From bot_config side, if the process fails, we catch the [os_err](https://chrome-internal.googlesource.com/infradata/config/+/refs/heads/master/configs/chromium-swarm/scripts/flutter_mac_android_staging.py#157); if the file parse fails, we catch [io_err](https://chrome-internal.googlesource.com/infradata/config/+/refs/heads/master/configs/chromium-swarm/scripts/flutter_mac_android_staging.py#175) and [value_err](https://chrome-internal.googlesource.com/infradata/config/+/refs/heads/master/configs/chromium-swarm/scripts/flutter_mac_android_staging.py#179). Do we still miss anything?

Reply:
The reasoning behind printing or logging the output is that if something goes unexpectedly we can still see the output from /var/log/system or log system in mac even after the file was deleted from the bot.

Reply:
Added. 
Also fixed a corner case of the getDeviceProperties.